src/solidity-frontend/solidity_grammar.h

```diff
@@ -264,6 +271,16 @@ enum ImplicitCastTypeT
 ImplicitCastTypeT get_implicit_cast_type_t(std::string cast);
 const char *implicit_cast_type_to_str(ImplicitCastTypeT type);
 
+enum VisibilityT
```

Comment:
Can you add some documentation here?

Reply:
Yes, I will add some documentation to explain.